AngularLanguageService.2022/AngularLanguageService.2022.csproj

```diff
@@ -1,100 +1,101 @@
-        <OutputPath>bin\Release\</OutputPath>
-        <DefineConstants>TRACE;VS2022</DefineConstants>
-        <ErrorReport>prompt</ErrorReport>
-        <WarningLevel>4</WarningLevel>
-    </PropertyGroup>
-    <ItemGroup>
-        <Compile Include="LanguageServer\AngularComponentToContentTypeProvider.cs" />
-        <Compile Include="LanguageServer\DiagnosticFormatterMiddleLayer.cs" />
-        <Compile Include="Properties\AssemblyInfo.cs" />
-    </ItemGroup>
-    <ItemGroup>
-        <None Include="package-lock.json" />
-        <None Include="package.json" />
-        <None Include="manifest.json" />
-        <None Include="source.extension.vsixmanifest">
-            <SubType>Designer</SubType>
-        </None>
-    </ItemGroup>
-    <ItemGroup>
-        <Content Include="../EULA.rtf" Link="EULA.rtf">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-            <IncludeInVSIX>true</IncludeInVSIX>
-        </Content>
-    </ItemGroup>
-    <ItemGroup>
-        <!-- MicroBuild signing -->
-        <PackageReference Include="Microsoft.VisualStudio.Internal.MicroBuild.VisualStudio" Version="2.0.65">
-            <PrivateAssets>all</PrivateAssets>
-            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-        </PackageReference>
-        <!-- LSP dependencies -->
-        <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Client" Version="17.0.5158" />
-        <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol" Version="17.0.5133" />
-        <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.0.63" />
-        <!-- For accessing WebTools' type definitions -->
-        <PackageReference Include="Microsoft.WebTools.Languages.Html.Editor" Version="17.0.633" />
-        <!-- Some dependencies indirectly depend on older versions of Newtonsoft.Json so we explicitly set it here. -->
-        <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
-    </ItemGroup>
-    <ItemGroup>
-        <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.3.2088">
-            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-            <PrivateAssets>all</PrivateAssets>
-        </PackageReference>
-    </ItemGroup>
-    <ItemGroup>
-        <!-- Signing -->
-        <FilesToSign Include="$(OutDir)\AngularLanguageService.2022.dll">
-            <Authenticode>Microsoft400</Authenticode>
-            <StrongName>StrongName</StrongName>
-        </FilesToSign>
-    </ItemGroup>
-    <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-    <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />
-    <Import Project="../AngularLanguageService.Shared/AngularLanguageService.Shared.projitems" Label="Shared" />
-    <Import Project="../VsixExtension.targets" />
+  <PropertyGroup>
```

Comment:
Does this file only include changes to formatting?

Reply:
Yeah and the addition of the new middle layer class. IDK why this keeps happening because I swear I don't care about how `.csproj` files are formatted lol